### PR TITLE
Signup and signin

### PR DESCRIPTION
**Pull request by iamalhera**
sign-in
sign-up

### CODE REVIEW

File: src/pages/signin/signin.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+.form-signin{
+    border : 2px solid var(--ca-primary-color-lighter);
+    background-color: var(--ca-light-bg-color);
+    width : 400px ;
+    border-radius: 5px;
+    font-family: 'Montserrat', sans-serif;
+}
+.form-header{
+    font-size : 2.0rem ;
```
Comment (FarhanMobashir) [inline src/pages/signin/signin.css:24]: make variable for typography variable as it will be easy to refactor when you will need to.

**Review by FarhanMobashir — COMMENTED** [review]
suggested some changes

**Review by Sahil25061999 — APPROVED** [review]
require some minor changes like using rem instead of px at some places.  The forgot password is wrapping in the next line, you can use whitespace: nowrap for that.

File: src/pages/signin/signin.css (hunk 1 above)
Comment (iamalhera) [reply src/pages/signin/signin.css:24]: I used a h4 tag having some styling in it, wanted to add a custom css, so added that

**Pull request merged**